docs-ref-conceptual/includes/query-azure-cli-concepts.md

```diff
@@ -0,0 +1,655 @@
+```
+
+### [PowerShell](#tab/powershell)
+
+```powershell-interactive
+az vm list --resource-group QueryDemo --query "sort_by([].{Name:name, Size:storageProfile.osDisk.diskSizeGb}, &Size)" --output table
+```
+
+### [Cmd](#tab/cmd)
+
+```cmd
+az vm list --resource-group QueryDemo --query "sort_by([].{Name:name, Size:storageProfile.osDisk.diskSizeGb}, &Size)" --output table
+```
+
+---
+
+```json
+Name     Size
+-------  ------
+Test-2   30
+Test-VM  32
+WinTest  127
+```
+
+See the [JMESPath specification - Built-in Functions](http://jmespath.org/specification.html#built-in-functions) for the full list of functions.
+
+## Formatting query results
+
+The Azure CLI uses JSON as its default output format, however different output formats may better suit a query depending on its purpose and results. Note that queries are always run on the `JSON` output first and then formatted.
+
+This section will go over `tsv` and `table` formatting and some use cases for each format. For more information about output formats, see [Output formats for Azure CLI commands](/cli/azure/format-output-azure-cli).
+
+### TSV output format
+
+The `tsv` output format returns tab- and newline-separated values without additional formatting, keys, or other symbols. This is useful when the output is consumed by another command.
+
+One use case for `tsv` formatting is queries that retrieve a value out of a CLI command, such as an Azure resource ID or resource name, and store the value in a local environment variable. By default the results are returned in JSON format. This may be an issue when dealing with JSON strings which are enclosed in `"` characters. The quotes may __not__ be interpreted by the shell if the command output is directly assigned to the environment variable. This can be seen in the following example that assigns a query result to an environment variable:
+
+### [Bash](#tab/bash)
+
+```azurecli-interactive
+USER=$(az vm show --resource-group QueryDemo --name TestVM --query 'osProfile.adminUsername')
+echo $USER
+```
+
+### [PowerShell](#tab/powershell)
+
+```powershell-interactive
+$USER=$(az vm show --resource-group QueryDemo --name TestVM --query "osProfile.adminUsername")
+echo $USER
+```
+
+### [Cmd](#tab/cmd)
+
+```cmd
+FOR /f %i IN ('az vm show --resource-group QueryDemo --name TestVM --query "osProfile.adminUsername"') DO SET USER=%i
+echo %USER%
+```
+
+---
+
+```json
+"azureuser"
+```
+
+To prevent enclosing return values with type information use `tsv`formatting as demonstrated in the following query:
+
+### [Bash](#tab/bash)
+
+```azurecli-interactive
+USER=$(az vm show --resource-group QueryDemo --name TestVM --query 'osProfile.adminUsername' --output tsv)
+echo $USER
+```
+
+### [PowerShell](#tab/powershell)
+
+```powershell-interactive
+$USER=$(az vm show --resource-group QueryDemo --name TestVM --query "osProfile.adminUsername" --output tsv)
+echo $USER
+```
+
```

Comment:
Should remove extra line here.

Reply:
sounds good, I've removed the extra line in the most recent version